.env-sample

```diff
@@ -1,2 +1,3 @@
 SNYK_TOKEN=replaceme
-COVERALLS_REPO_TOKEN=replaceme
\ No newline at end of file
+COVERALLS_REPO_TOKEN=replaceme
+MONGO_IMAGE_TAG=replaceme with the latest tag from https://hub.docker.com/r/nhsuk/mongodb-pharmacy/tags/
```

Comment:
Sorry, just one small change - new line.

Could you add http://eslint.org/docs/rules/eol-last to the eslint rules so we pick these up in the githooks.